### PR TITLE
Encryption Tools pack

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Encryption Tools pack

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/EncryptionTools/TestPlaybooks/playbook-test-encryption-decryption.yml [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      entryID:
+        simple: '${.=(val.InfoFile instanceof Array ? val.InfoFile[val.InfoFile.length-1].EntryID
+          : val.InfoFile.EntryID)}'
```
Comment (yaakovi) [inline Packs/EncryptionTools/TestPlaybooks/playbook-test-encryption-decryption.yml:423]: transformers are easier...
image

File: Packs/EncryptionTools/Integrations/EncryptionTools/EncryptionTools_test.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+class TestGetPublicKey:
+    @staticmethod
+    def test_no_data():
+        with pytest.raises(Exception) as e:
+            get_public_key()
+
+        assert e
+        assert e.type == DemistoException
+        assert 'Public key is not defined.' in str(e)
```
Comment (yaakovi) [inline Packs/EncryptionTools/Integrations/EncryptionTools/EncryptionTools_test.py:17]:
```suggestion
        with pytest.raises(DemistoException, match='Public key is not defined.'):
            get_public_key()
```

**Review by yaakovi — APPROVED** [review]
nice work.

update integration image.

File: Tests/conf.json [hunk 3]
```diff
@@ -3395,6 +3395,10 @@
             "playbookID": "CarbonBlackLiveResponseCloud-Test",
             "fromversion": "5.5.0",
             "is_mockable": false
+        },
+        {
+            "integrations": "EncryptionTools",
+            "playbookID": "playbook-test-encryption-decryption"
```
[the remark this answers is not in the document]
Comment (teizenman) [reply Tests/conf.json:3401]: Will remove it if we decide to go community here.

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

File: Packs/RSAEncryptionTools/Integrations/RSAEncryptionTools/RSAEncryptionTools.yml [hunk 4]
```diff
@@ -0,0 +1,115 @@
+name: RSA encryption Tools
```
Comment (yaakovi) [inline Packs/RSAEncryptionTools/Integrations/RSAEncryptionTools/RSAEncryptionTools.yml:1]:
```suggestion
name: RSA Encryption Tools
```

**Review by yaakovi — APPROVED** [review]

**Comment by Itay4** [comment]
@yaakovi what is the status here ?

**Comment by yaakovi** [comment]
stale PR, closing...
we will revisit if required.

**Pull request closed without merging**